transform1234/alt-frontend
# Added css in Admin School,student and teacher module

## Description

Added css in Admin School,student and teacher Form module

### Changes

Css Change

## How to test

Login with admin credentials and go to any student/teacher/school form and check the new css applied

Please consider using the closing keyword if the pull request is proposed to
fix an issue already created in the repository
(https://help.github.com/articles/closing-issues-using-keywords/)
